app/templates/admin/administer-user-modal.pug

```diff
@@ -256,6 +256,42 @@ block modal-body-content
                 button.btn.btn-primary.edit-prepaids-info-btn(data-prepaid-id=prepaid.id) Edit
     hr
 
+  h3.m-t-3#credits Add Extra Credits
+  #credit-form.form
+
+    if view.state === 'creating-credits'
+      .progress.progress-striped.active
+        .progress-bar(style="width: 100%")
+
+    else if view.state === 'made-credits'
+      h3 Credits created!
+      .alert.alert-success Credits created!
+
+    else
+      div(style="display: flex; gap: 20px; align-items: flex-start")
+        +license-form-group('Number of Credits')(id="credit-input" type="number", name="addCredits", min="1")
+          h4.small(style="max-width: 700px") Credits end at 11:59pm PT of the end date selected
+          +license-form-group('End Date')(type="date" name="endDate" value=view.momentTimezone().tz(view.timeZone).add(1, 'year').format('YYYY-MM-DD'))
+        .form-group
+          - creditType = view.creditType
+          label
+            span Credit Types
+            =":"
+            #credits-type-select
```

Comment:
let's use a dropdown instead. we don't want it to occupy more space as we add more options